apps/src/templates/sectionsRefresh/SetUpSectionForm.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moduleStyles from './sections-refresh.module.scss';
+import i18n from '@cdo/locale';
+
+export default function SetUpSectionForm({sectionNum, section, updateSection}) {
+  return (
+    <div>
+      <h2>{i18n.classSectionNum({num: sectionNum})}</h2>
+      <label>
+        {i18n.className()}
+        <input
+          type="text"
+          className={moduleStyles.classNameTextField}
+          value={section.name}
+          onChange={e => updateSection('name', e.target.value)}
+        />
+      </label>
+      <hr />
+    </div>
```

Comment:
this presumably won't matter if we're doing ajax submit buttons, but I noticed there's not a `<form>` here. was that an intentional decision? also, if we were to add a `<form>` to this page, I guess it'd have to go at the `SectionsSetUp` level, which would make this component having `Form` in the name misleading.

Reply:
> I noticed there's not a <form> here. was that an intentional decision?

Yes, at least for now.

> I guess it'd have to go at the SectionsSetUp level, which would make this component having Form in the name misleading.

Good point! Moving `Form` out of the name of this component might be a good idea so we can remain flexible